packages/components/src/mobile/bottom-sheet/cell.native.js

```diff
@@ -223,11 +224,22 @@ class BottomSheetCell extends Component {
 				styles.cellValue,
 				styles.cellTextDark
 			);
-			const finalStyle = {
+			const textInputStyle = {
 				...cellValueStyle,
 				...valueStyle,
 				...styleRTL,
 			};
+			const placeholderTextColor = disabled
+				? this.props.getStylesFromColorScheme(
+						styles.placeholderColorDisabled,
+						styles.placeholderColorDisabledDark
+				  ).color
+				: styles.placeholderColor.color;
```

Comment:
I finally calculated the placeholder text color to handle the different cases.

Reply:
This is great. I noticed this when I started my review and then noted you updated this code. :) The raw hex values stood out to me even when working on the [Clear Button issue](https://github.com/WordPress/gutenberg/pull/37402) from a while ago -- thanks for addressing them here!